subprojects/language-scala/src/main/java/org/gradle/language/scala/tasks/AbstractScalaCompile.java

```diff
@@ -185,8 +184,7 @@ private void configureIncrementalCompilation(ScalaCompileSpec spec) {
     @Incubating
     @Classpath
     public FileCollection getEffectiveAnnotationProcessorPath() {
```

Comment:
We could try removing this one or at least deprecate it.

Reply:
Same for `JavaCompile`, I presume?

Reply:
Yep

Reply:
Done in 590bb439451.